pkg/services/notifications/notifications.go

```diff
@@ -197,6 +197,7 @@ func (ns *NotificationService) SendEmailCommandHandlerSync(ctx context.Context,
 		AttachedFiles: cmd.AttachedFiles,
 		Subject:       cmd.Subject,
 		ReplyTo:       cmd.ReplyTo,
+		CustomHeaders: map[string]string{"slug": ns.Cfg.Slug},
```

Comment:
These are the only two spots where `buildEmailMessage()` gets called. Would be a good place to add tenant-level custom headers like slug.